src/platform_impl/linux/window.rs

```diff
@@ -101,17 +107,21 @@ impl Window {
       .insert(window_id);
 
     // Set Width/Height & Resizable
-    let win_scale_factor = window.scale_factor();
     let (width, height) = attributes
       .inner_size
       .map(|size| size.to_logical::<f64>(win_scale_factor as f64).into())
       .unwrap_or((800, 600));
-    window.set_default_size(1, 1);
-    window.resize(width, height);
+
+    let window_clone = window.clone();
+    glib::idle_add_local_once(move || {
+      window_clone.set_resizable(true);
+      window_clone.set_default_size(min_width, min_height);
+      window_clone.resize(width, height);
```

Comment:
resizing here makes the window create with a smaller size than expected, and then after a brief moment, it will jump into the requested size

Reply:
the initially created size is the correct window size. Changing the resizable property only adjusts the width so that the title is not truncated.

If the title is shortened, the size does not change. For example, if you change “Hit space to toggle resizability.” to “Hit space,” the size does not change after creation.